Tests/Features/Page/error.feature

```diff
@@ -0,0 +1,19 @@
+@mink:selenium2 @alice(Page) @alice(ErrorPage) @reset-schema
+Feature: Page not found
+
+Background:
+    Given I maximize the window
+    And I am on homepage
+
+  Scenario: I cannot acces a non exitant page
+    And I am on "/fr/imaginary-page"
+    Then I should see "404 not found"
```

Comment:
In debug mode, 404 Not Found is shown but not in debug false. We just need to add a Widget text with the content "404 Not Found"